internal/aws/k8s/k8sclient/endpoint.go

```diff
@@ -48,9 +48,6 @@ type EpClient interface {
 	PodKeyToServiceNames() map[string][]string
 	// Get the mapping between the service and the number of belonging pods
 	ServiceToPodNum() map[Service]int
-
-	// shutdown is only used internally by clientset to stop the EpClient
-	shutdown()
```

Comment:
I have tried to implement the interface outside of this package. But golang doesn't allow me to do that. That's why I have to split it into two interfaces. I also don't want to expose the `shutdown()` outside the package because I want to centralize the control in K8sClient.